sites/dwhdelft.nl/pages/chatgroups.vue

```diff
@@ -81,7 +83,15 @@ export default {
         .map((group) => {
           return { ...group, category: group.dir.substring(group.dir.lastIndexOf('/') + 1) }
         })
-        .sort((a, b) => (a.name > b.name ? 1 : -1)),
+        .sort(function (a, b) {
+          if (a.show_first ^ b.show_first) {
+            return a.show_first ? -1 : b.show_first ? 1 : 0
+          }
+          if (a.show_last ^ b.show_last) {
+            return a.show_last ? 1 : b.show_last ? -1 : 0
+          }
+          return a.name > b.name ? 1 : -1
+        }),
```

Comment:
Hm niet echt sold op specifieke properties voor first/last. Misschien een order property toevoegen?

Reply:
Hm, leek me wel chill om de rest gwn alfabetisch te laten sorten. Of bedoel je dus een order property met 1 voor wat nu fiets is en iets van -1 voor last (dat ze beide in 1 attribute zitten?)